e2e/service_metadata_test.go

```diff
@@ -51,49 +54,40 @@ var _ = Describe("creating a Service/Endpoint/EndpointSlice for a Tenant with ad
 					"clastix.io/custom-annotation": "buzz",
 				},
 			},
-		},
-	}
-	epsCR := &rbacv1.ClusterRole{
```

Comment:
Taking advantage of the additional role bindings offered with latest versions of Capsule, using the `endpointslice-controller`.